schema/crdb/11.0.0/up01.sql

```diff
@@ -0,0 +1,23 @@
+/*
+ * Drop the entire metric producer assignment table.
```

Comment:
It might be worth mentioning that we're doing this because we want to add a _required_ column and we don't have a great way to backfill the data -- _but_ we also don't need to because the data will be immediately repopulated.